yt/data_objects/construction_data_containers.py

```diff
@@ -871,6 +871,12 @@ def _fill_sph_particles(self, fields):
 
         bounds, size = self._get_grid_bounds_size()
 
+        period = self.ds.coordinates.period.copy()
+        if hasattr(period, "in_units"):
+            period = period.in_units("code_length").d
+        # TODO maybe there is a better way of handling this
+        periodic = any(self.ds.periodicity)
```

Comment:
since boolean value is never used directly... I'd also suggest a minor variable name change
```suggestion
        is_periodic = int(any(self.ds.periodicity))
```

Reply:
Done.